types/artifacts.d.ts

```diff
@@ -698,6 +704,33 @@ declare global {
         observedSpeedIndexTs: number;
       }
 
+      export interface Form {
+        /** If attributes is missing that means this is a formless set of elements. */
+        attributes?: { id: string, name: string, autocomplete: string, devtoolsNodePath: string, snippet: string,};
+        inputs: { id: string; nodeName: string; name: string; placeholder: string; autocomplete: string; devtoolsNodePath: string; snippet: string;}[];
+        labels: { id: string; nodeName: string; for: string; devtoolsNodePath: string; snippet: string;}[];
+      }
+
+      /** Attributes collected for every input element in the inputs array from the forms interface. */
+      export interface FormInputsType {
```

Comment:
```suggestion
      export interface FormInput {
```

since all of these are types so we tend not to suffix them with `Type` :)